ocp_resources/llama_stack_distribution.py

```
@@ -1,48 +1,59 @@
 # Generated using https://github.com/RedHatQE/openshift-python-wrapper/blob/main/scripts/resource/README.md
 
-
-from typing import Any
-
 from ocp_resources.resource import MissingRequiredArgumentError, NamespacedResource
 
+from typing import Any
+from ocp_resources.resource import NamespacedResource
```

Ruff (0.15.0)

[error] Redefinition of unused `NamespacedResource` from line 3: `NamespacedResource` redefined here

Remove definition: `NamespacedResource`

(F811)